e2e_samples/parking_sensors/README.md

```diff
@@ -209,15 +212,37 @@ More resources:
 
 ## How to use the sample
 
-### Prerequisites
+This sample has certain prerequisites for deployment, which will be detailed below. Please note that using the development container is optional. However, we recommend using the development container because it streamlines the setup process, ensures consistency across environments, and simplifies dependency management.
 
-1. [Github account](https://github.com/)
+### Setup
+
+Following the setup prerequisites, permissions, and deployment environment options.
+
+#### Prerequisites
+
+1. [Github account](https://github.com/) : If you do not have already, create your GitHub Account.
 2. [Azure Account](https://azure.microsoft.com/en-us/free/)
```

Comment:
For consistency, I would suggest adding the same sentence we have for GH: `If you do not have one already, create an Azure account`